src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const uniqueProperties = new Set(sourceString.split(';'));
+
+  uniqueProperties.delete('\n');
+  uniqueProperties.delete('\n  ');
+  uniqueProperties.delete('\n\n  ');
```

Comment:
It may seem like easy solution, but what if you'll have more non-valid elements?
You need to find a different way to get rid of useless spaces and new lines.